app/models/orchestration.rb

```diff
@@ -35,12 +35,12 @@ def run
   # rubocop:enable Rails/SkipsModelValidations,Metrics/CyclomaticComplexity
 
   def self.run(kind: :bootstrap, params: nil)
-    raise OrchestrationOngoing unless runnable?
+    raise OrchestrationOngoing if runnable?
     Orchestration.create!(kind: kind, params: params).tap(&:run)
   end
 
   def self.runnable?
-    Orchestration.in_progress.empty?
+    Orchestration.last.try(:status) == "in_progress"
```

Comment:
this then should be `! Orchestration.last.try(:status).in_progress?`

Reply:
`Orchestration.last.try(:in_progress?)` works fine as well